donate/routes.py

```diff
@@ -143,8 +143,10 @@ def donation():
             params['email'])
         app.logger.debug("Charge created: {}".format(charge_data))
     except StripeError as error:
-        app.logger.error("StripeError: ")
-        flash(error)
+        err = error.json_body.get('error', {})
```

Comment:
It is possible (and in fact the default case for the `StripeError` class, see [error.py, line 13](https://github.com/stripe/stripe-python/blob/f6adc3ea47ba9ebd5d57432c1ca33f0117661d6a/stripe/error.py#L13) and line 31) that `json_body` is `None` which will result in an error (`AttributeError: 'NoneType' object has no attribute 'get'`).  See previous comment for how to better handle errors.